sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -32,7 +33,6 @@ import org.apache.spark.sql.execution.datasources.{CreateTable, DataSource}
 import org.apache.spark.sql.types.StructType
 import org.apache.spark.storage.StorageLevel
 
-
```

Comment:
We usually avoid this kind of unrelated change.

Reply:
@viirya Ok!